secure/README.md

```diff
@@ -0,0 +1,34 @@
+# Securing Dataproc
+
+This initialization action installs the MIT distribution of the Kerberos packages, and configure Yarn, HDFS, Hive, Spark to integrate with Kerberose, which enables in-transit data encryption.
+It will also encrypt HTTP traffic (include Web UIs and shuffle) with SSL.
+
+## Using this initialization action
+You can use this initialization action to create a secured Dataproc cluster:
+  Using the `gcloud` command to create a new cluster with this initialization action.
+
+```bash
+gcloud dataproc clusters create <CLUSTER_NAME> \
+    --scopes cloud-platform \
+    --initialization-actions gs://dataproc-initialization-actions/secure/secure.sh \
+    --metadata "kms-key-uri=projects/<PROJECT_ID>/locations/global/keyRings/my-key-rings/cryptoKeys/my-key" \
```

Comment:
Should we parameterize key ring and key names?

Reply:
I believe a key-uri is enough to identify the key.